src/Menu/Menu.js

```diff
@@ -26,6 +33,127 @@ export default function Menu(props) {
         );
     });
 
+    const mainMenu = (
+        <div
+            className={cs("mainMenu", {
+                hidden: gameData.isMenuHidden,
+            })}
+        >
+            <button
+                className="fight"
+                disabled={gameData.isOppTurn}
+                onClick={attackMenu}
+            >
+                FIGHT
+            </button>
+            <button
+                className="pkmn"
+                disabled={gameData.isOppTurn}
+                onClick={changePokemon}
+            >
+                PKMN
+            </button>
+            <button
+                className="itemMenu"
+                disabled={gameData.isOppTurn}
+                onClick={item}
+            >
+                ITEM
+            </button>
+            <button className="run" disabled={gameData.isOppTurn} onClick={run}>
+                RUN
+            </button>
+        </div>
+    );
+
+    const itemMenu = (
+        <div
+            className={cs({
+                hidden: gameData.isItemMenuHidden,
+            })}
+        >
+            <ul className="itemList">
+                <li className="item">
+                    Potion <p>x3</p>
+                </li>
+                <li className="item">
+                    Pokeball <p>x6</p>
+                </li>
+                <li className="item">
+                    Berry <p>x3</p>
+                </li>
+                <li className="item">
+                    Silk Scarf <p>x1</p>
+                </li>
+                <li className="item">
+                    Revive <p>x1</p>
+                </li>
+                <li className="item">
+                    Super Potion <p>x3</p>
+                </li>
+            </ul>
+        </div>
+    );
+
+    const PKMNmenu = (
+        <div
+            className={cs({
+                hidden: gameData.isPartyMenuHidden,
+            })}
+        >
+            <ul className="partyList">
+                <li className="partyItem">
+                    <img
+                        className="sprite"
+                        src={charmanderSprite}
+                        alt="sprite img"
```

Comment:
Since this is not happening programmatically you should hard code the alt values to something morre specific than just sprite img. Maybe say which pokemon sprite each one is